crypto/ts/ts_rsp_verify.c

```diff
@@ -251,35 +250,12 @@ static int ts_check_signing_certs(PKCS7_SIGNER_INFO *si,
  err:
     if (!ret)
         TSerr(TS_F_TS_CHECK_SIGNING_CERTS,
-              TS_R_ESS_SIGNING_CERTIFICATE_ERROR);
+              ESS_R_ESS_SIGNING_CERTIFICATE_ERROR);
```

Comment:
wrong change

Reply:
Why?
Inside TS there are no more R_ESS_ definitions

Reply:
Because, `TSErr` will look in `TS_str_reasons` array, to pick the corresponding error message,
So, in this case, it will probably  found nothing or a wrong message,